docker-compose.aws.yml

```diff
@@ -0,0 +1,208 @@
+version: '3'
```

Comment:
Can we move this to `aws` dir ?

Reply:
move [aws/docker-compose.aws.yml](https://github.com/hyperdxio/hyperdx/pull/79/files#diff-3ed4dd5c300fc5bb5bc4fd260f8aaf90ed1da8866624166faba32a421aefb27e)